src/main/java/com/college/erpmgmt/entities/Student.java

```diff
@@ -1,19 +1,17 @@
 package com.college.erpmgmt.entities;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+
+import javax.persistence.*;
 import java.io.Serializable;
 
 @Entity
 @JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
 public class Student implements Serializable {

```

Comment:
these days, there are quite many libraries which saves us from writing the pojo classes.
few options are 

1. open-api with automatic code generation (a very good solution for automatic api documentation swagger generation and pojo classes generation - need some learning and understanding)
2. project lombok - focuses mainly on reducing the redundant code of getters and setters. very easy too use and reduces the code size (https://projectlombok.org/)

both of them are very good option for learning, you can use either or both in a project. i would suggest start with lombok.